src/test/java/org/jenkinsci/plugins/workflow/multibranch/JobPropertyStepTest.java

```diff
@@ -82,15 +97,19 @@
         */
         StepConfigTester tester = new StepConfigTester(r);
         properties = tester.configRoundTrip(new JobPropertyStep(properties)).getProperties();
-        assertEquals(1, properties.size());
-        assertEquals(ParametersDefinitionProperty.class, properties.get(0).getClass());
-        ParametersDefinitionProperty pdp = (ParametersDefinitionProperty) properties.get(0);
+        assertEquals(2, properties.size());
+        ParametersDefinitionProperty pdp = getPropertyFromList(ParametersDefinitionProperty.class, properties);
+        assertNotNull(pdp);
         assertEquals(1, pdp.getParameterDefinitions().size());
         assertEquals(BooleanParameterDefinition.class, pdp.getParameterDefinitions().get(0).getClass());
         BooleanParameterDefinition bpd = (BooleanParameterDefinition) pdp.getParameterDefinitions().get(0);
         assertEquals("flag", bpd.getName());
         assertTrue(bpd.isDefaultValue());
-        assertEquals(Collections.emptyList(), tester.configRoundTrip(new JobPropertyStep(Collections.<JobProperty>emptyList())).getProperties());
+
+        // TODO The fact that there's *always* a PipelineTriggersJobProperty is a bit weird. Is that the right approach?
```

Comment:
That was the design, since it is not an `OptionalJobProperty`, right? I mean, you could delete the property when empty I suppose. But you would need to be careful to null out `WorkflowJob.triggers` if you are not already doing so.